src/components/accordion/AccordionMenu.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FC, PropsWithChildren } from 'react';
+
+import { AccordionProps } from '@chakra-ui/react';
```

Comment:
is it needed for this component to have full flexibility or can we restrict the props to some subset?